UnityProject/AddressablePackingProjects/SoundAndMusic/.vsconfig

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "version": "1.0",
+  "components": [ 
+    "Microsoft.VisualStudio.Workload.ManagedGame"
+  ]
+} 
```

Comment:
should this have been committed?

Reply:
Hmmm, I really don't know.... but I don't think it's causing issues?

Reply:
it shouldn't but it is not causing issues either